tests/framework/session/test_session_registration_hooks.py

```diff
@@ -142,15 +142,15 @@ def test_register_pipelines_with_duplicate_entries(
         self, tmp_path, mock_package_name, mock_pipeline
     ):
         session = KedroSession.create(mock_package_name, tmp_path)
-        context = session.load_context()
+        session.load_context()
```

Comment:
I'm likely missing something - why do we need this line?

Reply:
Good spot we don't need it 😄